### DISCUSSION TITLE
Divide-x and flex-row-reverse unexpected result

### DISCUSSION THREAD

**Discussion by alexander-schranz** [tailwindlabs/tailwindcss#8298] [Help]
**What version of Tailwind CSS are you using?** v3.0.24
**What browser are you using?** Firefox
**What operating system are you using?** macOS
**What build tool (or framework if it abstracts the build tool) are you using?**  webpack
**What version of Node.js are you using?** 14
**Reproduction URL** https://play.tailwindcss.com/HZOMdegUJF

**Describe your issue**

If you reverse the flex items the border will not longer be between the elements.

**Comment by adamwathan** [comment]
Hey! Please read the docs: https://tailwindcss.com/docs/divide-width#reversing-children-order

**Reply by alexander-schranz** [reply]
Didn't see that thank you :)
